cmd/lakectl/cmd/root.go

```diff
@@ -598,6 +598,9 @@ func initConfig() {
 	if cfgFile != "" {
 		// Use config file from the flag.
 		viper.SetConfigFile(cfgFile)
+	} else if envCfgFile, exists := os.LookupEnv("LAKECTL_CONFIG_FILE"); exists && envCfgFile != "" {
```

Comment:
```suggestion
	} else if envCfgFile := os.LookupEnv("LAKECTL_CONFIG_FILE"); envCfgFile != "" {
```
if the file doesn't exist, envCfgFile will be empty anyway